packages/mobile/src/send/Send.tsx

```diff
@@ -100,8 +100,17 @@ export const sendScreenNavOptions = ({
 }: {
   route: RouteProp<StackParamList, Screens.Send>
 }) => {
-  const goQr = () => navigate(Screens.QRNavigator)
-  const title = route.params?.isRequest
+  const isOutgoingPaymentRequest = route.params?.isOutgoingPaymentRequest
+
+  const goToQRScanner = () =>
```

Comment:
Navigating directly to the scanner with a param that lets us know if it's for a payment request